### DISCUSSION TITLE
Welcome (and some thoughts on discussions vs. issues)

### DISCUSSION THREAD

**Discussion by daveaglick** [statiqdev/Discussions#1] [Ideas]
👋 Hi everyone!

I'm really excited to start using GitHub Discussions as a way to promote, grow, and extend our little community.

I suspect one of the bigger questions around discussions will be how to use it compared to the issue trackers in the individual Statiq repositories. I'm sure we'll learn about and refine that distinction over time, but my initial thought is that discussions should be for questions, ideas, showcasing, etc. while issues are for features and bugs. I.e., issues are "work to be done" while discussions are not (at least not initially - though an idea in a discussion thread could certainly turn into a discussion).

**Comment by adrwh** [comment]
How about a tutorial, to walk "beginner" users through setting up a minimal layout, theme, with some use of short codes etc?

**Reply by daveaglick** [reply]
I've wanted to do a series of video tutorials _for years_ going back to the early Wyam days. Hopefully I'll find some time as Statiq reaches maturity - I agree something like that would be great for helping folks get up to speed quickly.

**Comment by punker76** [comment]
@daveaglick Yeah! Congrats to be allowed for testing out this thing. IMO this here should not be an issue tracker for the individual sites, more a general help board like you already said. Greetz
